src/sinks/logdna.rs

```diff
@@ -101,7 +101,8 @@ impl HttpSink for LogdnaConfig {
     type Input = serde_json::Value;
     type Output = Vec<BoxedRawValue>;
 
-    fn encode_event(&self, event: Event) -> Option<Self::Input> {
+    fn encode_event(&self, mut event: Event) -> Option<Self::Input> {
+        self.encoding.apply_rules(&mut event);
```

Comment:
Should we apply these rules after we pull off the fields below?

Reply:
Good question... I think we should not because below we change fields for the sink. If we move, then it would be possible to change the already build event?
TODO: need to check all `apply_rules`.. :disappointed:

Reply:
Reviewed all sinks and made changes where it was required.